apps/gatsby/src/AppConfig/AppConfig.js

```diff
@@ -336,6 +357,24 @@ export class AppConfig extends React.Component {
                 type: "text",
               }}
             />
+            <TextField
+              name="previewWebhookUrl"
+              id="previewWebhookUrl"
+              labelText="Preview Webhook URL"
+              value={this.state.previewWebhookUrl}
+              onChange={this.updatePreviewWebhookUrl}
+              onBlur={this.validatePreviewWebhookUrl}
+              className={styles.input}
+              helpText="Optional Preview Webhook URL. If provided, your site will be automatically rebuilt as you change draft content."
```

Comment:
This might be a bit misleading since the user has to make a click to start a preview build. Maybe something like "Including this will allow you to create preview builds on Gatsby cloud as you draft content"?